src/utils/interfaces/LayerManager.js

```diff
@@ -0,0 +1,153 @@
+/* eslint import/no-unresolved: 0 */
+/* eslint import/extensions: 0 */
+
+import L from 'leaflet';
+
+export default class LayerManager {
+
+  // Constructor
+  constructor(map, onLayerAddedOK = null, onLayerAddedKO = null) {
+    this._map = map;
+    this._mapLayers = {};
+    this._onLayerAddedOK = onLayerAddedOK;
```

Comment:
Let's use Success and Error instead of ok and ko, like we do in the actions. Consistency...kill me